dockerfiles/pytorch-dev/install_rocm_from_tarballs.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# -----------------------------------------------------------------------------
+# install_rocm_from_tarballs.sh
+#
+# Download and install ROCm tarballs for specified AMDGPU targets from
+# TheRock's nightly GitHub releases.
+#
+# Usage:
+#   INSTALL_PREFIX=/pathToTheRock VERSION=6.4.0rc20250420 ./install_rocm_from_tarballs.sh "gfx942 gfx1100"
+#
+# Environment Variables (optional):
+#   VERSION           - Full version like 6.4.0rc20250420 (required if no version.json)
+#   ROCM_VERSION      - Base version like 6.4.0 (optional, auto-extracted from VERSION or version.json)
+#   RELEASE_TAG       - GitHub release tag (default: nightly-release)
+#   ROCM_VERSION_DATE - Build date (default: 1 day ago)
+#   INSTALL_PREFIX    - Installation path (default: /therock/build/dist/rocm)
+#   OUTPUT_ARTIFACTS_DIR - Downloaded tarballs location (default: /rocm-tarballs)
+#
+# Requirements:
+#   curl (auto-installed if missing)
+#   jq (auto-installed if missing)
+#   bash
+# -----------------------------------------------------------------------------
+
+# Ensure required tools
+for tool in curl jq; do
+  if ! command -v "$tool" >/dev/null 2>&1; then
+    echo "[INFO] $tool not found. Installing..."
+    if command -v apt-get >/dev/null 2>&1; then
+      apt-get update
+      apt-get install -y "$tool"
+    else
+      echo "[ERROR] $tool installation not supported on this OS. Please install it manually."
+      exit 1
+    fi
+  fi
+done
+
+# Configuration
+RELEASE_TAG="${RELEASE_TAG:-nightly-release}"
+ROCM_VERSION_DATE="${ROCM_VERSION_DATE:-$(date -d '1 days ago' +'%Y%m%d')}"
+INSTALL_PREFIX="${INSTALL_PREFIX:-/therock/build/dist/rocm}"
+OUTPUT_ARTIFACTS_DIR="${OUTPUT_ARTIFACTS_DIR:-/rocm-tarballs}"
+GITHUB_RELEASE_BASE_URL="https://github.com/ROCm/TheRock/releases/download"
```

Comment:
It could be helpful to put the organization name and repository name in one or two variables. That way, someone testing releases or producing their own downstream releases in a fork wouldn't need to edit the script source to point at a different path.

Reply:
Sounds good -  separated them